pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
So the latest travis issue with Tesseract is `java.lang.UnsatisfiedLinkError: Error looking up function 'TessBaseAPICreate': /usr/lib/libtesseract.so.3.0.2: undefined symbol: TessBaseAPICreate`. When I run `apt-get install tesseract-ocr` on my local Ubuntu 16.04 system, it installs the same package, but with `libtesseract.so.3.0.4`, which contains the API calls for Tess4J that we used, which are present in Tesseract 3.0.3+. It looks like the Travis system is running Ubuntu 12.04LTS, which [ends support at 3.0.2](https://github.com/tesseract-ocr/tesseract/wiki/Compiling#leptonica). How do we want to proceed with this knowledge?